src/index.html

```diff
@@ -13,6 +13,46 @@
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card__product-image"
+        src="./images/imac.jpeg"
+        alt="product-image"
+      />
+
+      <h1 class="card__name-of-product">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h1>
```

Comment:
This tag used for title of the page, so you need to use the other tag here and fix class
```suggestion
      <h2 class="card__title">
        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
      </h2>
```